examples/gno.land/p/demo/markdown/table.gno

```diff
@@ -0,0 +1,106 @@
+package markdown
+
+import (
+	"strings"
+
+	"gno.land/p/demo/ufmt"
+)
+
+// Table defines the structure for a markdown table
+type Table struct {
+	header []string
+	rows   [][]string
+}
+
+// ValidateColumns checks if the number of columns in each row matches the number of columns in the header
+func (t *Table) ValidateColumns() error {
+	numCols := len(t.header)
+	for _, row := range t.rows {
+		if len(row) != numCols {
+			return ufmt.Errorf("row %v does not match header length %d", row, numCols)
+		}
+	}
+	return nil
+}
+
+// NewTable creates a new Table instance, ensuring the header and rows match in size
+func NewTable(header []string, rows [][]string) (*Table, error) {
```

Comment:
What is the advantage of having a NewTable that returns a Stringer? Wouldn't it be enough to just have a `func Table`?

Reply:
would be convenient to have this helper or maybe a second one that doesn't take `string` but `Stringer` or `interface{}`; or maybe we can keep this for a standalone p/.

Reply:
I personally enjoy this usage, which is compatible with `ufmt`: https://pkg.go.dev/text/tabwriter

Reply:
not sure i understood your idea correctly, plz take a look at the last commit changes